packages/ra-core/package.json

```diff
@@ -57,10 +57,10 @@
         "final-form": "^4.18.5",
         "inflection": "~1.12.0",
         "lodash": "~4.17.5",
-        "node-polyglot": "^2.2.2",
         "prop-types": "^15.6.1",
         "query-string": "~5.1.1",
         "ra-language-english": "^3.0.0-alpha.4",
+        "ra-i18n-polyglot": "^3.0.0-alpha.4",
```

Comment:
Shouldn't this be in the dev dependencies only, for tests ? Otherwise we are still including polyglot for all users

Reply:
Just like for the english language, we need a sensible default.